### ISSUE TITLE
CLI STDOUT output

### ISSUE THREAD

**Issue by rochacbruno** [usebruno/bruno#829]
Hi,

Maybe I am missing something, but I was not able to find the answer, I am doing.

```console 
❯ cat settings.bru 
meta {
  name: settings
  type: http
  seq: 1
}

get {
  url: http://localhost:5001/api/automation-hub/_ui/v1/settings/
  body: none
  auth: none
}

❯ bru run settings.bru
Running Request 

settings (200 OK) - 93 ms

Requests:    1 passed, 1 total
Tests:       0 passed, 0 total
Assertions:  0 passed, 0 total
Ran all requests - 93 ms
```

And that is great for tests.

But I am trying to reuse the same `.bru` files I have to chain requests on the CLI and I see I can pass `--output` to have it saved to a file, however I wanted to have output send to the STDOUT so I can '|' to other application such as `jq` is that possible?

Something like:

```console
bru run settings.bru --output stdout | jq
{

 ...
}
```

**Comment by jzorn** [comment]
Yeah, I'd like to use Bruno in some of my pipelines, it would be really useful to have have this feature. In my opinion, if the --format flag is provided but no --output flag, stdout would be a reasonable target.

**Comment by dernorberto** [comment]
+1
to script and pipe directly to jq for processing

**Comment by rochacbruno** [comment]
I am closing this issue as this is stale and I have no interest in Bruno anymore (using another client) thanks.
